src/main/kotlin/eu/europa/ec/eudi/pidissuer/port/input/IssueCredential.kt

```diff
@@ -302,12 +301,10 @@ private val log = LoggerFactory.getLogger(IssueCredential::class.java)
  * Usecase for issuing a Credential.
  */
 class IssueCredential(
```

Comment:
I don't like the removal of clock.

We are relying too much on the dependency injection. 
Anyways, don't change it.